src/main/java/io/jenkins/plugins/coverage/BuildUtils.java

```diff
@@ -0,0 +1,17 @@
+package io.jenkins.plugins.coverage;
+
+import hudson.model.Run;
+
+public class BuildUtils {
+	public static Run<?, ?> getPreviousNotFailedCompletedBuild(Run<?, ?> b) {
```

Comment:
In Jenkins project we usually spaces instead of tabs for spacing. It's totally up to you, but tabs may be preferable for other contributors